src/components/assets/styles/variables.ts

```diff
@@ -34,6 +34,16 @@ const vars = {
 
   switchShadow:
     '0 0.1875rem 0.5rem rgba(0, 0, 0, 0.15), 0 0.1875rem 0.0625rem rgba(0, 0, 0, 0.06)',
+
```

Comment:
@vidhya-metacell do we need this if we do the MUI theming?

Reply:
@zsinnema Yes, as with other style properties. User can change the color/shadows whatever as required by the project.

Reply:
an instead of doing this let the user create a theme